app/src/main/java/com/mghelas/internship_playground/AllProductsActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.mghelas.internship_playground;
+
+import android.os.Bundle;
+import android.app.Activity;
+import android.view.Gravity;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.mghelas.internship_playground.sweetsfactory.Chocolate;
+
+public class AllProductsActivity extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_all_products);
+
+        LinearLayout productsLl = findViewById(R.id.productsLl);
+        LinearLayout.LayoutParams layoutParams = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.WRAP_CONTENT);
+        layoutParams.setMargins(30, 20, 30, 0);
```

Comment:
why do you set up manually margins from code and do not set up them in xml?

Reply:
I set margins to "product" linear layout which I create dynamically